.github/workflows/publish-to-dockerhub.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+name: Publish to Docker Hub
+
+on:
+  release:
+    types: [released]
+
+env:
+  TOKEN: ${{ secrets.GH_TOKEN }}
+
+jobs:
+  publish-to-dockerhub:
+    name: Publish to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 50
```

Comment:
I let that in place, because it was like this before, but I don't know if it makes sense to fetch so deep